presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreenEvent.kt

```diff
@@ -1,8 +1,8 @@
 package me.androidbox.presentation.agenda.screen
 
-import java.time.LocalDate
+import java.time.ZonedDateTime
 
 sealed interface AgendaScreenEvent {
-    data class OnDateChanged(val date: LocalDate) : AgendaScreenEvent
+    data class OnDateChanged(val date: ZonedDateTime) : AgendaScreenEvent
     data class OnShowDropdown(val shouldOpen: Boolean) : AgendaScreenEvent
```

Comment:
This one, I'd change the name

OnShowDropdown tells me that it's an event that's going to be triggered once a dropdown is shown
Which is not the case

Maybe something like
"ShowDropdown" and "HideDropdown"
or
"ChangeDropdownStatus" if you want to use just 1 event with a boolean

Reply:
@MKiperszmid I went with name OnChangeDropdownStatus